openbb_terminal/stocks/options/op_helpers.py

```diff
@@ -257,10 +261,11 @@ def get_greeks(
     if not all(
         col in chain_columns for col in ["strike", "impliedVolatility", "optionType"]
     ):
-        console.print(
-            "[red]It's not possible to calculate the greeks without the following "
-            "columns: `strike`, `impliedVolatility`, `optionType`.\n[/red]"
-        )
+        if "delta" not in chain_columns:
```

Comment:
This was tricky to me. Tradier provides all the greeks.  Intrinio only provides delta.  So my take was to show the exact data we get if its either of those sources. And delta was the common column.  Any other thoughts?

Reply:
I think the line `if "delta" not in chain.columns:` in `options_view.py` would already avoid the greeks to be calculated, but let's keep it as a double check in case this function gets called from any other place.